src/cloudai/parser.py

```diff
@@ -71,24 +81,64 @@ def parse(
         except TestConfigParsingError:
             exit(1)  # exit right away to keep error message readable for users
 
-        logging.debug(f"Parsed {len(tests)} tests: {[t.name for t in tests]}")
-        test_mapping = {t.name: t for t in tests}
+        if not PLUGIN_ROOT.exists():
+            logging.debug(f"PLUGIN_ROOT path '{PLUGIN_ROOT}' does not exist. Plugins will not be enabled.")
```

Comment:
IMO second part of this message is misleading. If Scenario has hooks, but they don't exist, we will see a failure.

Let's also rename `PLUGIN_ROOT` constant.

Reply:
> IMO second part of this message is misleading. If Scenario has hooks, but they don't exist, we will see a failure.

Updated

> Let's also rename PLUGIN_ROOT constant.

What do you mean by this?

Reply:
PLUGIN --> HOOK

Reply:
Updated.